.github/workflows/ubuntu-20.04.yml

```diff
@@ -0,0 +1,212 @@
+---
```

Comment:
When merging your branch with mine, we should make the empty lines consistent. I prefer you style here with an empty line between sequential steps and two empty lines between jobs.